apps/bsp/bulk-scan-processor/processor-test-image-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: bulk-scan-processor-processor-test
+spec:
+  imageRepositoryRef:
+    name: bulk-scan-processor
```

Comment:
Can this reflect to the new update - `bulk-scan-processor-processor-test`

Reply:
Update already reflected in the metadata - can you clarify?

Reply:
Script updated and reapplied.